igcollect/sfp_dom.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env python3
+"""igcollect - SFP Digital Optical Monitoring metric collector
+
+Copyright (c) 2022 InnoGames GmbH
+
+This script reads SFP EEPROM and parses it in order to collect optical
+diagnostics data.
+
+To properly understand what happens on this script, a read of the SFF-8472
+standard is highly recommended.
+
+In some places, we gave up on making things Pythonic in detriment of shaving
+off a few ms on execution time.
```

Comment:
Is this super crucial? Then I will check a bit more over it.

Reply:
Actually, not. I did some rudimentary benchmarking and the difference was negligible, so I removed them. I will remove this part.